nixos/modules/virtualisation/containers.nix

```diff
@@ -564,6 +570,18 @@ in
               '';
             };
 
+            writableAPIVFS = mkOption {
```

Comment:
Is the thing we're making writeable the `API_VFS`? Or should the env var be parsed as

> [ "SYSTEMD_NSPAWN_API" "VFS" "WRITABLE" ]

Reply:
good question :grinning:, but I think that in this case it really is the `API_VFS` that is being made writable as none of the other `SYSTEMD_NSPAWN` environment variables have `API` in them.